src/draw_parameters/mod.rs

```diff
@@ -432,6 +436,33 @@ impl<'a> From<&'a AnySamplesPassedQuery> for SamplesQueryParam<'a> {
     }
 }
 
+/// Specifies the depth offset applied to rendered geometry
+#[derive(Debug, Copy, Clone)]
+pub struct PolygonOffset{
```

Comment:
Please put a space between t and {. Also please put spaces between the type name and variable name. Thank you.